packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -233,6 +233,7 @@ describe('Call Tests', () => {
         getAudioTracks: jest.fn().mockReturnValue([mockTrack]),
       },
       on: jest.fn(),
+      setUserMuted: jest.fn(),
```

Comment:
We can remove the outputStream object from mockStream as we have removed that logic in the src file.

Reply:
removed